graders/c/Dockerfile

```diff
@@ -1,7 +1,7 @@
 FROM ubuntu:24.04
-
 LABEL maintainer="jonatan@yorku.ca"
 
+ARG CACHEBUST=1
```

Comment:
I'd be somewhat inclined to put this as the first line in files for consistency.

Reply:
I would prefer it directly below the `FROM label` but I'll make that change.